docs/mergeduty/howto.md

```diff
@@ -184,7 +185,7 @@ Run the bump-esr [no-op trial run](#do-migration-no-op-trial-runs), and show the
 
 ```sh
 export version=68
-export full_version=68.3.0
+export full_version=68.4.0
```

Comment:
maybe we should template this? Rather than bump *docs* each time?

Reply:
Thought of it too, but I was also hoping that by the next time we perform mergeduty, we may have some progress towards automation, hence that'll be coming from tree or so.